frontends/concrete-python/concrete/fhe/version.py

```diff
@@ -0,0 +1,5 @@
+"""
```

Comment:
currently, version.txt is 

__version__ = "0.0.0"

simply, so will the comment disappear? Will it be a problem there is no docstring here?

Reply:
It's okay, docstring is only there to satisfy `make pcc`.